ci/cov

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I'd much prefer to having `.sh` suffixes in shell scripts, it saves quite some time navigating in projects ;)

Reply:
This is an experiment of something I will move to beaver, so becoming a *command* where the suffix makes no sense.